deploy-renku/entrypoint.sh

```diff
@@ -28,8 +28,9 @@ fi
 
 NAMESPACE_EXISTS=$( curl -s -H "Authorization: Bearer $RENKUBOT_RANCHER_BEARER_TOKEN" \
       -X GET \
-      "${RANCHER_DEV_API_ENDPOINT}/namespaces?projectId=${RANCHER_PROJECT_ID}"| grep $RENKU_NAMESPACE)
-if [[ ! -n $NAMESPACE_EXISTS ]]; then
+      -d "projectId=${RANCHER_PROJECT_ID}" \
```

Comment:
Are you sure that this way of providing a request parameter works? If I remember correctly, when providing the project id as request data, it is simply ignored.

Reply:
You are right. I missed this because things still work and the grep command has to filter more stuff. I will switch this part back the way it was.